sympy/parsing/tests/test_c_parser.py

```diff
@@ -1327,6 +1329,47 @@ def test_binary_operators():
             '}'
         )
 
+        # Unary operators as a part of binary operators is not supported yet!
+        c_src_raise1 = (
+            'void func()'+
+            '{' + '\n' +
+            'int a;' + '\n' +
+            'a = -1;' + '\n' +
+            '}'
+        )
+
+        c_src_raise2 = (
+            'void func()'+
+            '{' + '\n' +
+            'int a;' + '\n' +
+            'a = -+1;' + '\n' +
+            '}'
+        )
+
+        c_src_raise3 = (
+            'void func()'+
+            '{' + '\n' +
+            'int a;' + '\n' +
+            'a = 2*-2;' + '\n' +
+            '}'
+        )
+
+        # Given token ? is not implemented yet!
```

Comment:
This comment is in the wrong place

Reply:
Since given comment applies for c_src_raise4 as well as c_src_raise5, I needed a generic symbol for `{}` (See error message) , hence I used `?`(but that is raised in c_src_raise6 as you said). I will change `?` to `{}` then.

Reply:
I know, I explained it in a really bad manner. In simple words, `?` meant `<insert some token here>`, but little did I think, it is also a part of conditional operator! :)

Reply:
It's just really confusing instead of helping people. I think it'll be better to just remove the comment. Also, can you squash the commits again after removing the comment?

Reply:
> It's just really confusing instead of helping people. I think it'll be better to just remove the comment.

Agreed, looks like it is confusing

>Also, can you squash the commits again after removing the comment?

Alright